lib/main_micro_alp.pm

```diff
@@ -276,17 +276,19 @@ sub load_slem_on_pc_tests {
     if (get_var('PUBLIC_CLOUD_DOWNLOAD_TESTREPO')) {
         load_publiccloud_download_repos();
     } else {
+        # SLEM basic test
         loadtest("boot/boot_to_desktop");
         loadtest("publiccloud/prepare_instance", run_args => $args);
         loadtest("publiccloud/registration", run_args => $args);
         # 2 next modules of pubcloud needed for sle-micro incidents/repos verification
         loadtest("publiccloud/transfer_repos", run_args => $args);
         loadtest("publiccloud/patch_and_reboot", run_args => $args);
 
+        loadtest("publiccloud/slem_basic", run_args => $args);
```

Comment:
with such change , taking into account that we have nothing except `slem_basic` running anywhere I would just delete ALL lines below this one as for now . Later we will add them back but this will require more if/else which would be more clear how better to do in further PRs

Reply:
done

Reply:
Ah but bering last module I have to set publiccloud_multi_module = 0. Doing